internal/deepdoc/parser/pdf/table/table_merge.go

```
@@ -124,3 +140,66 @@ func MergeTablesAcrossPages(tables []pdf.TableItem, medianHeights map[int]float6
+	prevMaxY := 0.0
+	for _, g := range grids {
+		if len(g) == 0 {
+			continue
+		}
+		minY, maxY := gridYExtent(g)
+		if prevMaxY > 0 {
+			// Place this page's rows below everything stacked so far, with a
+			// gap of at least one row height to avoid false row grouping.
+			shift := prevMaxY - minY + math.Max(maxY-minY, 1)
+			g = shiftGridY(g, shift)
+			minY += shift
```

golangci-lint (2.12.2)

[error] ineffectual assignment to minY

(ineffassign)